app/preload.js

```diff
@@ -16,15 +16,24 @@ function navigate(url) {
 
 ipcRenderer.on('isPlaying', (event) => {
   const isPlaying = !!document.querySelector('.playing')
-  event.sender.send('isPlaying', isPlaying)
+  const icon = document.getElementsByClassName('playbackSoundBadge__avatar sc-media-image')[0].children[0].children[0].getAttribute('style')
+  var urlStart = icon.indexOf('url("') + 5;
+  var urlEnd = icon.indexOf('");') - urlStart;
+  var iconURL = icon.substr(urlStart, urlEnd)
```

Comment:
Can't say I am a regex expert. To be honest I didn't think of future proofing, I just stopped when it worked ha ha. This makes a lot of sense though. You're a smart man.